tests/test_trainer.py

```diff
@@ -34,6 +34,11 @@ def output_logs_dir():
         yield temp_dir
 
 
+@pytest.fixture('session')
+def catdog_num_classes():
+    return 2
+
+
```

Comment:
Can you add a test that passes in an override for `num_gpus`? A copy-paste of one of the existing should be fine.